dev-docs/source/MantidUsedIconsTable.rst

```diff
@@ -0,0 +1,130 @@
++===============+============================+=====================+
+| Copy          | "mdi.content-copy"         | |copy|              |
++---------------+----------------------------+---------------------+
+| Paste         | "mdi.content-paste"        | |paste|             |
++---------------+----------------------------+---------------------+
+| Cut           | "mdi.content-cut"          | |cut|               |
++---------------+----------------------------+---------------------+
+| Settings      | "mdi.settings"             | |settings|          |
++---------------+----------------------------+---------------------+
+| Eraser        | "mdi.eraser"               | |eraser|            |
++---------------+----------------------------+---------------------+
+| Delete        | "mdi.trash-can"            | |delete|            |
++---------------+----------------------------+---------------------+
+| Error         | "mdi.asterisk"             | |error|             |
++---------------+----------------------------+---------------------+
+| Plot          | "mdi.chart-line"           | |plot|              |
++---------------+----------------------------+---------------------+
+| Plot Multiple | "mdi.chart-areaspline"     | |plot-multiple|     |
++---------------+----------------------------+---------------------+
+| Up            | "mdi.arrow-up"             | |up|                |
++---------------+----------------------------+---------------------+
+| Down          | "mdi.arrow-down"           | |down|              |
++---------------+----------------------------+---------------------+
+| Play          | "mdi.play"                 | |play|              |
++---------------+----------------------------+---------------------+
+| Stop          | "mdi.stop"                 | |stop|              |
++---------------+----------------------------+---------------------+
+| Pause         | "mdi.pause"                | |pause|             |
++---------------+----------------------------+---------------------+
+| Plus          | "mdi.plus"                 | |plus|              |
++---------------+----------------------------+---------------------+
+| Close         | "mdi.close"                | |close|             |
++---------------+----------------------------+---------------------+
+| Search        | "mdi.folder"               | |search|            |
++---------------+----------------------------+---------------------+
+| Refresh       | "mdi.twitter-retweet"      | |refresh|           |
++---------------+----------------------------+---------------------+
+| Home          | "mdi.home"                 | |home|              |
++---------------+----------------------------+---------------------+
+| Pan/Move      | "mdi.arrow-all"            | |pan|               |
++---------------+----------------------------+---------------------+
+| Zoom In       | "mdi.magnify-plus-outline" | |zoom|              |
```

Comment:
Why is there no Zoom Out listed? Do we not actually use that anywhere?

Reply:
We currently don't, in a plot window, we use home to `reset` where the zoom is.

Reply:
This is out of scope here, but why do we use a different icon to Matplotlib here?

The functionality is the same from what I can tell and in Matplotlib this is shown with a single magnifying glass (as the tool in question can be used to both zoom in and out).

Reply:
I think this is the default layout used for matplotlib-qt windows. It may be better to ask whoever implemented chose/implemented that window's usage.

Reply:
(@martyngigg)